src/reporting/tezos/TezosConseilClient.ts

```diff
@@ -296,7 +296,43 @@ export namespace TezosConseilClient {
             maps.push({definition, content});
         }
 
-        return { contract: accountID, maps };
+        return { contract, maps };
+    }
+
+    /**
+     * Returns a value for a given key. The big map id is either provided as a paramter or the smallest of the possibly multiple big maps associated with the given contract address is used. Note that sometimes key values must be wrapped in quotes for key types that are non-numeric and not byte type.
+     * 
+     * @param serverInfo Conseil server connection definition.
+     * @param key Key to query for.
```

Comment:
We should specify whether this is encoded or not.

Reply:
docs were updated